helm/flowforge/templates/emqx.yaml

```diff
@@ -0,0 +1,240 @@
+                        ]
+                        enable = true
+                        enable_authn = true
+                        mountpoint = "${client_attrs.team}"
+                        max_connections = infinity
+                        acceptors = 16
+                        proxy_protocol = false
+                        proxy_protocol_timeout = 3s
+                        tcp_options {
+                            backlog = 1024
+                            send_timeout = 15s
+                            recbuf = 2KB
+                            sndbuf = 4KB
+                            buffer = 4KB
+                            high_watermark = 1MB
+                            nodelay = true
+                            reuseaddr = true
+                            keepalive = "none"
+                        }
+                    }
+                }
+                ssl {
+                    default {
+                        enable = false
+                    }
+                }
+                wss {
+                    default {
+                        enable = false
+                    }
+                }
+                ws {
+                    default {
+                        bind = "0.0.0.0:8080"
+                        access_rules = [
+                            "allow all"
+                        ]
+                        enable = true
+                        enable_authn = true
+                        mountpoint = "${client_attrs.team}"
+                        max_connections = infinity
+                        proxy_protocol = false
+                        proxy_protocol_timeout = 3s
+                        tcp_options {
+                            backlog = 1024
+                            send_timeout = 15s
+                            recbuf = 2KB
+                            sndbuf = 4KB
+                            buffer = 4KB
+                            high_watermark = 1MB
+                            nodelay = true
+                            reuseaddr = true
+                            keepalive = "none"
+                        }
+                        websocket {
+                            mqtt_path = "/"
+                            allow_origin_absence = true
+                            check_origin_enable = false
+                            fail_if_no_subprotocol = true
+                            supported_subprotocols = "mqtt, mqtt-v3, mqtt-v3.1.1 mqtt-v5"
+                            mqtt_piggyback = multiple
+                            compress = false
+                            idle_timeout = 7200s
+                            max_frame_size = infinity
+                            proxy_address_header = "x-forwarded-for"
+                            proxy_port_header = "x-forwarded-port"
+                        }
+                    }
+                }
+            }
+            dashboard {
+                default_password = topSecret
```

Comment:
Can this be linked to the value from a Secret?

Reply:
I don't think so as it's a direct embed into the config file, the idea is that you should change it on first login.

The defaults are to not expose the dashboard that uses this password externally, but it could be accessed internally to the cluster (e.g. I am currently doing kubectl port forwarding)

Reply:
FYI It seems like it is possible to do it via environmental variables `EMQX_DASHBOARD__DEFAULT_PASSWORD` for the password and `EMQX_DASHBOARD__DEFAULT_USERNAME`

Reply:
OK, will need to test if we can inject individual env vars and not wipe out the ones already set on the pods.

Reply:
Changes, made, but will need to test